code/workspaces/web-app/src/config/storage.js

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+
+let cachedData;
+
+async function storageConfig() {
+  if (cachedData) {
+    return cachedData;
+  }
+
+  const { data } = await axios.get('/storage_config.json');
+  cachedData = data;
+  return cachedData;
+}
+
+export async function storageTypes() {
+  const data = await storageConfig();
+  return Object.entries(data.types)
+    .map(([key]) => key);
```

Comment:
Could this be simplified to `return Object.keys(data.types)`?

Reply:
Thanks, good idea